tests/Fixer/ControlStructure/NoEmptyBlockFixerTest.php

```diff
@@ -0,0 +1,815 @@
+        ];
+
+        yield 'alternate if with side effect in body' => [
+            '<?php if ($foo): echo 1; endif;',
+        ];
+
+        yield 'alternate if with side effect in braces' => [
+            '<?php ',
+            '<?php if ($foo->bar()): endif;',
+        ];
+
+        yield 'alternate if without side effect' => [
+            '<?php ',
+            '<?php if ($foo): endif;',
+        ];
+
+        yield 'alternate if without side effect but comment in body' => [
+            '<?php if ($foo): /* todo */ endif;',
+        ];
+
+        yield 'alternate if without side effect but doc comment in body' => [
+            '<?php if ($foo): /** todo */ endif;',
+        ];
+
+        yield 'alternate if without side effect with comments' => [
+            '<?php /*1*//*2*//*3*//*4*//*5*/',
+            '<?php /*1*/if/*2*/(/*3*/$foo/*4*/): endif;/*5*/',
+        ];
+
+        yield 'alternate if without side effect with doc comments' => [
+            '<?php /**1*//**2*//**3*//**4*//**5*/',
+            '<?php /**1*/if/**2*/(/**3*/$foo/**4*/): endif;/**5*/',
+        ];
+
+        yield 'alternate if with side effect in elseif body' => [
+            '<?php if ($foo): elseif ($bar): baz(); endif;',
+        ];
+
+        yield 'alternate if with side effect in elseif braces' => [
+            '<?php ',
+            '<?php if ($foo): elseif ($foo->baz()): endif;',
+        ];
+
+        yield 'alternate if without side effect and elseif without side effect' => [
+            '<?php ',
+            '<?php if ($foo): elseif ($bar): endif;',
+        ];
+
+        yield 'alternate if without side effect and elseif without side effect with comments' => [
+            '<?php /*1*//*2*//*3*//*4*//*5*//*6*//*7*//*8*/',
+            '<?php /*1*/if/*2*/(/*3*/$foo/*4*/): elseif/*5*/(/*6*/$bar/*7*/): endif;/*8*/',
+        ];
+
+        yield 'alternate if without side effect and elseif without side effect with doc comments' => [
+            '<?php /**1*//**2*//**3*//**4*//**5*//**6*//**7*//**8*/',
+            '<?php /**1*/if/**2*/(/**3*/$foo/**4*/): elseif/**5*/(/**6*/$bar/**7*/): endif;/**8*/',
+        ];
+
+        yield 'alternate elseif without side effect but comment in body' => [
+            '<?php if ($foo): elseif ($bar): /* todo */ endif;',
+        ];
+
+        yield 'alternate elseif without side effect but doc comment in body' => [
+            '<?php if ($foo): elseif ($bar): /** todo */ endif;',
+        ];
+
+        yield 'alternate multiple elseif without side effect' => [
+            '<?php ',
+            '<?php if ($foo): elseif ($bar): elseif ($baz): elseif ($boz): endif;',
+        ];
+
+        yield 'alternate multiple elseif one with side effect' => [
+            '<?php if ($foo): elseif ($bar): foo(); elseif ($baz): elseif ($boz): endif;',
+        ];
+
+        yield 'alternate if with side effect in else' => [
+            '<?php if ($foo): else: bar(); endif;',
```

Comment:
fyi
i remember we had the discussion that we may stop touching any file that is using alternative syntax one day

Reply:
Yes, but there are many other fixers with test cases for alternative syntax so I think we can leave it here too. If we finally decide to drop such support, we can then remove it from all fixers.

Reply:
it's more like "consider to not bother about it next time"